pkg/arvo/sys/vane/ames.hoon

```diff
@@ -4027,13 +4312,13 @@
             =.  peer-state    (update-peer-route her peer-state)
             =.  metrics.keen  on-timeout:fi-gauge
             =^  want=(unit want)  wan.keen
-              (pop-left:fi-deq wan.keen)
+              (fall (mole |.([`val.head rest]:(pop:fi-mop wan.keen))) `wan.keen)
```

Comment:
I don't think we should need to virtualize this.  Jumping in and out of a road here could slow things down, so let's look for an alternative that doesn't need virtualization.

Reply:
That should be less expensive than a road.